running-a-nats-service/configuration/README.md

```diff
@@ -347,20 +347,91 @@ jetstream {
+| `prev_encryption_key`                     |    The previous encryption key. Used when changing storage encryption keys. | (not set)   | 2.10.0   |
+| `max_outstanding_catchup` | Max in-flight bytes for stream catch-up                                                                                                                                                   | 64MB                    | 2.9.0   |
+| `max_buffered_msgs`                     |    Maximum number of messages Jetstream will buffer in memory when falling behind with RAFT or I/O. Use to protect against OOM when there are write burst to a queue. | 10.000  | 2.11.0   |
+| `max_buffered_size`                     |    Maximum number of bytes Jetstream will buffer in memory when falling behind with RAFT or I/O. Use to protect against OOM when there are write burst to a queue. | 128MB  | 2.11.0   |
+| `request_queue_limit`                     |    Limits the number of API commands Jetstream will buffer in memory. When the limit is reached clients will get error responses rather than timeout. Lower the value if you want to detect clients flooding Jetstream. | 10.000  | 2.11.0   |
 | `sync_interval`           | Examples: `10s` `1m` `always`  -   Change the default fsync/sync interval for page cache in the filestore. By default JetStream relies on stream replication in the cluster to guarantee data is available after an OS crash. If you run JetStream without replication or with a replication of just 2 you may want to shorten the fsync/sync interval. - You can force an fsync after each messsage with `always`, this will slow down the throughput to a few hundred msg/s.                                                                                                           | 2m                      | 2.10.0  |
+| `strict`                     |    Return errors for invalid Jetstream API requests. Some older client APIs may not expect this. Set to `false` for maximum backwards compatibility.  | `true`  | 2.11.0   |
+| `unique_tag`                     |    Jetstream peers will be placed in servers with tags unique relative to the `unique_tag`  prefix. E.g. nodes in a cluster (or supercluster) are tagged `az:1`,`az:1`,`az:2`,`az:2`,`az:3`,`az:3`,`az:3` . Setting `unique_tag=az` will result in a new replica 3 stream to be placed in all three availability zones.  | (not set))  | 2.8.0  |
+| `tpm`                     |  Trusted Platform Module   [TPM base encryption](#jetstream-tpm-encryption) | `tpm {}` (not set)  | 2.11.0   |
+| `limits`                     |   [Jetstream server limits](#jetstream-server-limits) | `limits{}`  (not set) | 2.8.0   |
 
 
+### JetStream Account Settings
+A Jetstream section may also appear in accounts. Jetstream is disabled by default. The minimal configuration will enable Jetstream.
+```text
+accounts {
+  A {}
```

Comment:
Small typo, need to drop the closing `}`.

Reply:
Thanks @MauriceVanVeen. Updated.